view/frontend/templates/button_product_page.phtml

```diff
@@ -148,6 +150,14 @@ if (!$block->isSupportableType()) return;
                     if (!hints) {
                         return false;
                     }
+
+                    var validator = jQuery( '#product_addtocart_form' ).validate();
```

Comment:
jQuery is required / included / encapsulated as $ in the script. Use it that way for consistency.

Reply:
Fixed, thank you